libs/perun/pipes/src/lib/perun-pipes.module.ts

```diff
@@ -22,7 +24,10 @@ import { GetResourceRoutePipe } from './get-resource-route.pipe';
     GroupSyncIconPipe,
     GroupSyncToolTipPipe,
     GroupSyncIconColorPipe,
-    GetResourceRoutePipe
+    GetResourceRoutePipe,
+    GroupSyncIconColorPipe,
+    MillisecondsFromEpochToDatePipe,
+    ServiceStateBlockedToStringPipe
```

Comment:
If you didnt used some of these pipes delete them

Reply:
I removed MillisecondsFromEpochToDatePipe